body/stretch_body/gamepad_teleop.py

```diff
@@ -208,9 +208,10 @@ def command_robot_joints(self, robot):
                 print("Switch D-Pad to Head Control")
                 self.do_single_beep(robot)
                 self.skip_x_button = True
-        # skip x button press by N cycles after a toggle 
+            self.time_since_dexwrist_switch = time.perf_counter()
+        # skip x button press for 1 second after a toggle 
         if self.skip_x_button:
-            if self._i % 10 == 0:
+            if (time.perf_counter() - self.time_since_dexwrist_switch) > 1:
```

Comment:
I recommend decreasing the wait time to 0.5 seconds, which is a better UX for aggressive users.
```suggestion
            if (time.perf_counter() - self.time_since_dexwrist_switch) > 0.5:
```